.vscode/extensions.json

```diff
@@ -0,0 +1,17 @@
+{
+  // See https://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
```

Comment:
I think these are highlighted in red because comments are not allowed in JSON

Reply:
This was auto-generated from the VSCode command.  Do you think we should remove it? I don't think it's hurting anything as far as VSCode is concerned (no errors or anything), but if you'd prefer it removed let me know.